.github/workflows/build-and-release-stable.yml

```diff
@@ -189,24 +193,40 @@ jobs:
     uses: meshery/meshery/.github/workflows/cncf-playground-deploy-meshery.yaml@master
     secrets: inherit
 
-  notify-via-email:
-    runs-on: ubuntu-latest
+  email-on-failure:
     needs: [update-rest-api-docs, update-graphql-docs, build, ctlrelease, call-dde-release-workflow, call-helm-chart-releaser, email-meshery-release-notes-workflow, call-cncf-playground-rollout]
-    if: ${{ always() && contains(needs.*.result, 'failure') }}
+    if: ${{ always() }}
+    runs-on: ubuntu-latest
     steps:
-      - name: Send email notification on failure
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "18"
+      - name: Find failed jobs.
+        run: |
+          cd .github/scripts/
+          curl https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/jobs >> workflow_run.json
+          node ./getFailedJobs.js >> output.txt
```

Comment:
Instead running a node js script we can simply parse the json data from curl using jq something like this
```suggestion
          curl "https://api.github.com/repos/${{ github.repository }}/actions/runs/${{ github.run_id }}/jobs" | jq -r '.jobs[] | select(.conclusion == "failure") | "\(.name): \(.steps[-1].name)"' > output.txt

```

Reply:
![image](https://github.com/meshery/meshery/assets/34866732/b37e210d-bdad-483a-9bca-bcc00fd547a6)
Working fine. Thanks for the advice @suhail34.